extensions/stac-eo-spec.md

```diff
@@ -49,15 +49,20 @@ there is no valid EPSG code.
 
 ### Band Object
 
-| element             | type info | description                                                                                                                                                   |
-| ------------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| name                | string    | The name of the band (e.g., "B01", "B02", "B1", "B5", "QA").                                                                                                  |
-| common_name         | string    | The name commonly used to refer to the band to make it easier to search for bands across instruments. See below for a list of accepted common names.          |
-| gsd                 | number    | Ground Sample distance, the nominal distance between pixel centers available, in meters. See eo:gsd for more information. Defaults to eo:gsd if not provided. |
-| accuracy            | number    | The expected error between the measured location and the true location of a pixel, in meters on the ground.                                                   |
-| center_wavelength   | number    | The center wavelength of the band, in microns.                                                                                                                |
-| full_width_half_max | number    | Full width at half maximum (FWHM). The width of the band, as measured at half the maximum transmission, in microns.                                           |
-
+| Field Name          | Type     | Description                                                  |
+| ------------------- | -------- | ------------------------------------------------------------ |
+| name                | string   | The name of the band (e.g., "B01", "B02", "B1", "B5", "QA"). |
+| common_name         | string   | The name commonly used to refer to the band to make it easier to search for bands across instruments. See below for a list of accepted common names. |
+| description         | string   | Description to fully explain the band. [CommonMark 0.28](http://commonmark.org/) syntax MAY be used for rich text representation. |
+| gsd                 | number   | Ground Sample distance, the nominal distance between pixel centers available, in meters. See `eo:gsd` for more information. Defaults to `eo:gsd` if not provided. |
+| accuracy            | number   | The expected error between the measured location and the true location of a pixel, in meters on the ground. |
+| center_wavelength   | number   | The center wavelength of the band, in microns.               |
+| full_width_half_max | number   | Full width at half maximum (FWHM). The width of the band, as measured at half the maximum transmission, in microns. |
+| resolution          | number   | Spatial resolution of the band, in meters.                   |
```

Comment:
How is this different then gsd?

Reply:
Thanks for spotting this mistake on my side while merging. Wasn't meant to be in here as it's not different. Removed.